c-pallets/file-bank/src/lib.rs

```diff
@@ -438,7 +446,7 @@ pub mod pallet {
 				T::Scheduler::contains_scheduler(sender.clone()),
 				Error::<T>::ScheduleNonExistent
 			);
-			let file_hash_bounded: BoundedString<T> =
+			let file_hash_bounded: BoundedString<T> = // Can be used Self::vec_to_bound instead?
 				file_hash.try_into().map_err(|_| Error::<T>::BoundedVecError)?;
```

Comment:
Based on your suggestion, I chose to use vec_to_bound:
https://github.com/CESSProject/cess/pull/88/files#diff-9c37b1a61a6e9394aac1e9383bf9d3fac9539378d1af8fcd75aed4f8a4f42967R461